eng/Packages.Data.props

```diff
@@ -166,7 +167,7 @@
     <PackageReference Update="Azure.ResourceManager.Storage" Version="1.4.2" />
 
     <!-- CDK -->
-    <PackageReference Update="Azure.Provisioning" Version="1.2.0" />
+    <PackageReference Update="Azure.Provisioning" Version="1.4.0-beta.1" />
```

Comment:
We generally don't want to have central versions point at betas.   Please override locally, if possible.  If this is going to impact a wide area of libraries, then override in your `/sdk` level `Directory.Build.Props` file.